accelerator/cli_info.h

```diff
@@ -48,6 +49,7 @@ typedef enum ta_cli_arg_value_e {
   CACHE,
   CONF_CLI,
   PROXY_API,
+  CHECK_PERIOD,
```

Comment:
Would it be better to use `health track` rather than `check`?

Reply:
sure